MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/MTDome-Troubleshooting/MTDome-Recover-and-clear-Faults.rst

```diff
@@ -0,0 +1,122 @@
+
+In other instances, the dome may reject move commands, when there's a subsystem ``FAULT``.
+
+.. _MTDome-Troubleshooting-MTDome-Recover-and-clear-Faults-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+- If a ``FAULT`` is indicated in the *Azimuth Control Software Status* (ACS status) window, the Dome has an internal ``FAULT`` condition.
+- Look at the Chronograf dashboard `MTDome Status`_ to verify the system status.
+
+.. Note::
+    
+    The Dome has several subsystems that continue to operate when a specific subsystem goes to ``FAULT``, the CSC itself won't go to ``FAULT``, unless it is an unrecoverable issue.
+    A subsystem going to ``FAULT`` is recoverable.
+
+..
+.. _MTDome-Troubleshooting-MTDome-Recover-and-clear-Faults-Procedure-Steps:
+
+Procedure Steps
+===============
+1. Disable the Dome following mode in *MTDomeTrajectory*.
+
+.. code-block:: python
+  :caption: :file:`run_command.py`
+
+    component: MTDomeTrajectory
+    cmd: setFollowingMode
+    parameters:
+        enable: false
+..
+
+2. To clear the subsystem ``FAULT``, use the MTDome **exitFault** configuration command via the :file:`run_command.py` SAL script. 
+In most cases, running the command once will clear the ``FAULT``, but you may need to repeat the process a few times.
+
+.. code-block:: python
+  :caption: :file:`run_command.py`
+
+    component: MTDome
+    cmd: exitFault
+..
+
+3. After executing the command, verify its success by checking the ACS status in the `MTDome Status`_ dashboard. 
+
+.. Note::
+    When the **exitFault** command is sent, the CSC executes the *resetDrivesAz* command before **exitFault**  command.
+
+.. code-block:: python  
+  :caption: :file:`run_command.py`
+
+    component: MTDome
+    cmd: resetDrivesAz
+    paramaters:
+      reset: [1,1,1,1,1]
+  
+..
+
+4. Now, at this point, you can try to move the Dome again. 
+
+- Confirm the Dome moves by slewing to a nearby position; in the example below, 45 degrees azimuth. 
+- Make sure that the Dome moves before ``ENABLE`` the *MTDomeTrajectory* mode.
+
+.. code-block:: python
+  :caption: :file:`run_command.py`
+    
+    component: MTDome
+    cmd: moveAz
+    parameters:
+        position: 45
+..
+
+5. Make sure you check the MTDome logs when trying to move it the first time after recovering it. 
+If you see the following warning message:
+
+.. warning::
+    Ignoring *moveAz* command for position=300.0 and velocity=0.0 because it is a duplicate command
+..
+
+The error means the CSC is ignoring the move command, regardless of whether the position mentioned in the message is the current position of the Dome.
+In this case, try to move the Dome to a different position.
+
+.. _MTDome-Troubleshooting-MTDome-Recover-and-clear-Faults-Post-Condition:
+
+Post-Condition
+==============
+- MTDome ``FAULT`` are clear and operations can continue.
```

Comment:
Here the verb fault is the one to use. "MTDome faults are clear and operations can continue." (Without the "-" at the beginning since there is not more list items.)